lib/athena_health/endpoints/patients.rb

```diff
@@ -168,6 +168,14 @@ def create_patient_document(practice_id:, department_id:, patient_id:, params: {
         )
       end
 
+      def create_patient_admin_document(practice_id:, department_id:, patient_id:, params: {})
+        @api.call(
```

Comment:
While it's not totally necessary, do we have any interest in expanding the "models" so that it returns a simple object or are we fine keeping our outputs as raw JSON and just processing that on the caller side?